### ISSUE TITLE
sources

### ISSUE THREAD

**Issue by ADM91** [nodebit/nodebit#14]
In my opinion:
-  it makes sense to take the user back to the main menu automatically after they have successfully created a data source. 
- There should be some way to quickly verify the source is correctly configured
- A source icon should only be created on the main menu if a source is submitted (now it is generated by clicking source)

**Comment by davidsongirard** [comment]
Agreed all around. I will be doing a pretty complete redesign and I would like to build these features in. I will keep you posted.
